test/e2e/specs/align-hook.test.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	insertBlock,
+	getEditedPostContent,
+	setCodeEditorContent,
+	switchToEditor,
+	getAllBlocks,
+	selectBlockByClientId,
+} from '../support/utils';
+import { activatePlugin, deactivatePlugin } from '../support/plugins';
+
+describe( 'Align Hook Works As Expected', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'gutenberg-test-align-hook' );
+	} );
+
+	beforeEach( async () => {
+		await newPost();
+	} );
+
+	afterAll( async () => {
+		await deactivatePlugin( 'gutenberg-test-align-hook' );
+	} );
+
+	const getAlignmentToolbarLabels = async () => {
+		const buttonLabels = await page.evaluate( () => {
+			const buttons = [];
+			document.querySelectorAll(
```

Comment:
You can use map here to avoid using mutations:
```js
const buttons = document.querySelectorAll(
    '.editor-block-toolbar button[aria-label^="Align"]'
).map(
    ( button ) => button.getAttribute( 'aria-label' )
);
```

Reply:
I tried to use a map before and tried again now, and I always get this error `Evaluation failed: TypeError: document.querySelectorAll(...).map is not a function`.
I think the map is not supported for this case as the object returned by querySelectorAll does not includes a map.

Reply:
You can do `[ ...document.querySelectorAll() ].map`